src/stash/protos/objects_v25.proto

```diff
@@ -15,6 +15,7 @@
 // migration using these two files, and no matter how they types change in the future, we'll always
 // have these snapshots to facilitate the migration.
 
+// buf breaking: ignore (does currently not require backward-compatibility)
```

Comment:
All of the `objects_X.proto` files shouldn't actually change at all, is that possible to enforce with buf?

Reply:
I [suggested to leave this out of scope for this PR](https://materializeinc.slack.com/archives/C01LKF361MZ/p1696499794191789?thread_ts=1696439050.873529&cid=C01LKF361MZ). This can be done as a follow up.

Reply:
Shall we still remove the ignore entry so that there is still at least the validation with `WIRE` level?

Reply:
Shall we remove the ignore entry so that there is still at least the validation with `WIRE` level?

Reply:
:+1: Yeah `objects_XX.proto` should not be ignored.

Reply:
✅ Removed them from the ignore list.

Reply:
Thanks @aalexandrov and @nrainer-materialize!